images/router/haproxy/conf/haproxy-config.template

```diff
@@ -57,8 +57,9 @@ global
   tune.maxrewrite 8192
   tune.bufsize 32768
 
-  # Prevent vulnerability to POODLE attacks
-  ssl-default-bind-options no-sslv3
+  # Configure the TLS versions we support
+  ssl-default-bind-options no-sslv3 ssl-min-ver {{env "SSL_MIN_VERSION" "TLSv1.0"}}
+    {{- if ne (env "SSL_MAX_VERSION" "") "" }} ssl-max-ver {{env "SSL_MAX_VERSION"}}{{ end }}
```

Comment:
This means it'll be impossible _not_ to use `ssl-min-ver` and (optionally) `ssl-max-ver` — is that okay? I'm not sure yet if the operator will want to leave in place any existing ingress controllers (and their router configurations, which lack a security profile in 4.1).

If we are going to forcibly migrate all existing ingress controllers with unspecified security profiles to Intermediate, then I see no issue removing the configurability from the template.

Reply:
`ssl-min-ver` is effectively already `TLSv1.0` because we have `no-sslv3` set.

I believe that the defaults I added for both match the current behavior.

Reply:
Changed the default to TLS 1.2 for https://bugzilla.redhat.com/show_bug.cgi?id=1746467